src/tigris.ts

```diff
@@ -119,13 +121,14 @@ const DEST_NAME_KEY = "destination-name";
 export class Tigris {
 	private readonly grpcClient: TigrisClient;
 	private readonly observabilityClient: ObservabilityClient;
-	private readonly config: TigrisClientConfig;
+	private readonly _config: TigrisClientConfig;
 
 	/**
 	 *
 	 * @param  {TigrisClientConfig} config configuration
 	 */
 	constructor(config?: TigrisClientConfig) {
+		dotenv.config();
```

Comment:
What's the behavior if there is no .env file?

Reply:
Doesn't next inject its own config variables through a different config file? What is the behavior in such cases?